src/views/forms/FormMosaicDefinitionTransaction/FormMosaicDefinitionTransaction.vue

```diff
@@ -33,7 +33,7 @@
             </div>
           </template>
         </FormRow>
-
+        <RentalFee :rental-type="'mosaic'" :rental-fee="formItems.rentalFees"></RentalFee>
```

Comment:
seems that `duration` is not being passed over to the component here?

Reply:
does duration affect fees while creating new mosaics?  or just while creating new root namespaces?